test/test_admin.py

```diff
@@ -414,7 +414,7 @@ def test_12_admin_user_search_json(self):
         # warning should be issued for non-found users
         data = {'user': 'nothingexists'}
         res = self.app_post_json('/admin/users', data=data)
-        warning = ("We didn't find")
+        warning = ("We didn&#39;t find")
```

Comment:
Missing i18n for that

Reply:
Yes! Let's merge this first, and then we can fix these other issues. This solution will be the path to avoid these errors, so we can have good translations.